### PR TITLE
Add client for AppVeyor REST API

### PR DESCRIPTION
**Pull request by topic2k**
https://www.appveyor.com/docs/api/

### CODE REVIEW

File: agithub/AppVeyor.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    >>> status, data = ci.api.projects[account_name][project_slug].get()
+    ... same as above
+
+    >>> status, data = ci.api.projects.topic2k.eventghost.buildcache.delete()
+    >>> status
+    ... 204
+
+    That's all there is to it. (blah.post() should work, too.)
+
+    NOTE: It is up to you to spell things correctly. A AppVeyor object
```
Comment (jpaugh) [inline agithub/AppVeyor.py]: Nitpick: should be "an AppVeyor"

**Review by jpaugh — APPROVED** [review]
Thanks for adding a new API! That's the best demonstration of agithub's power. I'd like to see the typo fixed, but that's a minor detail. I'll merge within a couple days.

**Comment by topic2k** [comment]
Should i squash the commits?

**Comment by jpaugh** [comment]
@topic2k No, that's fine. Thanks again!

**Pull request merged**